pydicom/tests/test_json.py

```diff
@@ -271,13 +272,14 @@ def test_sort_order(self):
         assert ds_json.index('"00100020"') < ds_json.index('"00100030"')
         assert ds_json.index('"00100030"') < ds_json.index('"00100040"')
 
-    def test_suppress_invalid_tags(self):
+    @mock.patch("pydicom.DataElement.to_json_dict", side_effect=ValueError)
+    def test_suppress_invalid_tags(self, _):
         """Test tags that raise exceptions don't if suppress_invalid_tags True.
         """
         ds = Dataset()
-        ds.add_new(0x00100010, 'PN', ['Jane^Doe'])
+        ds.add_new(0x00100010, 'PN', 'Jane^Doe')
```

Comment:
As this no longer raises an exception and I couldn't find another example of an exception, I just mocked it.

Reply:
`ds.add_new(0x00100010, "PN", 1234)` should do it